docs/application/native/guides/device/battery-monitor.md

```diff
@@ -1,26 +1,28 @@
 # Battery Monitor
 
-You can access the information related to the power consumption by applications or by hardware resources on a battery-powered device for a specific duration of time. The value is returned as the percentage of the total battery capacity.
+You can access the information related to the power consumption by applications or by hardware resources available on a battery-powered device for a specific duration of time. The value of the battery power usage is returned in the milliampere hour(mAh).
 
 The power consumption of an application is calculated by accumulating the usage of various individual resources used by the application.
 
 This feature is supported in wearable applications only.
 
-The Battery Monitor APIs provide following provisions to fetch the battery usage:
+The Battery Monitor APIs provide the following provisions to fetch the battery usage:
 
 - Getting battery usage information of an application.
 
-    You can [retrieve battery usage information of an application](#appusage_get) by specifying its application ID, [resource IDs](#resource_key), and the [time duration](#duration_key).
+    You can [retrieve battery usage information of an application](#appusage_get) by specifying its application ID, [resource IDs](#resource_key), and timestamps such as **from** and **to** as per Unix Epoch time format.
 
 - Getting battery usage information of a resource.
 
-    You can [retrieve battery usage information of single or multiple resources](#resourceusage_get) by specifying its [resource IDs](#resource_key), and the [time duration](#duration_key).
+    You can [retrieve battery usage information of single or multiple resources](#resourceusage_get) by specifying its [resource IDs](#resource_key), and timestamps such as **from** and **to** as per Unix Epoch time format.
+
+    You can retrieve the data that is recorded within the last seven days.
 
 ## Prerequisites
 
 To enable your application to use the Battery Monitor functionality:
 
-1. To use the [Battery Monitor](../../api/wearable/latest/group__CAPI__SYSTEM__BATTERY__BATTERY__MONITOR__MODULE.html) API, the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
+1. To use the [Battery Monitor](../../api/wearable/5.5/group__CAPI__SYSTEM__BATTERY__BATTERY__MONITOR__MODULE.html) API, the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
```

Comment:
@d1-agarwal  

latest is the sym link for 5.5 and is there any specific reason you changed it to 5.5.